Program.cs

```diff
@@ -1,4 +1,128 @@
-﻿using System;
-Console.WriteLine ("Hello, World!");
+﻿// --------------------------------------------------------------------------------------------
+// Training ~ A training program for interns at Metamation, Batch - July 2023
+// Copyright (c) Metamation India.                                              
+// ------------------------------------------------------------------------
+// Program.cs
+// MYLIST<T> 
+// Implement a custom MyList<T> class using arrays as the underlying data structure.
+// --------------------------------------------------------------------------------------------
+using System;
+// Test cases
+MyList<int> list = new ();
+list.Add (1);
+list.Add (2);
+list.Add (3);
+list.Add (4);
+list.Add (5);
+list.Add (6);
+list.Add (7);

```

Comment:
Instead of writing such repetitive statements, see if they can be simplified by loops